examples/systemToParaReserveTransferForeignAssets.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * When importing from @substrate/asset-transfer-api it would look like the following
+ *
+ * import { AssetsTransferApi, constructApiPromise } from '@substrate/asset-transfer-api'
+ */
+import { AssetsTransferApi, constructApiPromise } from '../src';
+import { TxResult } from '../src/types';
+import { GREEN, PURPLE, RESET } from './colors';
+
+/**
+ * In this example we are creating a call to send foreign asset '{"parents":"1","interior":{"X2":[{"Parachain":"2125"},{"GeneralIndex":"0"}]}}'
```

Comment:
This will be a teleport so the name of the file is misleading.

Reply:
In this example we are sending `TNKR` from assethub to Moonriver which I assumed should be a reserve transfer since it isn't native the destination. Is that incorrect?